nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/distribution/DistributionObject.java

```diff
@@ -41,7 +46,14 @@
     @Produces(MediaType.APPLICATION_JSON)
     public Object doGet() {
         Map<String, Serializable> map = new HashMap<>();
-        map.put("foo", "bar");
+        map.put(Environment.PRODUCT_NAME, PlatformVersionHelper.getApplicationName());
+        map.put(Environment.PRODUCT_VERSION, PlatformVersionHelper.getApplicationVersion());
+        map.put(Environment.DISTRIBUTION_NAME, PlatformVersionHelper.getDistributionName());
+        map.put(Environment.DISTRIBUTION_VERSION, PlatformVersionHelper.getDistributionVersion());
+        map.put(Environment.DISTRIBUTION_SERVER, PlatformVersionHelper.getDistributionHost());
+        map.put(Environment.DISTRIBUTION_DATE, PlatformVersionHelper.getDistributionDate());
+        map.put(Environment.BUNDLES, (Serializable) RuntimeInstrospection.getBundleIds());
+
```

Comment:
Why don't we have the bundle versions ?
The format should be:
```
    "nuxeo.osgi.bundles": [
        {
          "name": "org.nuxeo.ecm.core.convert",
          "version": "10.10-HF05" 
        },
        {
          "name": "org.nuxeo.ecm.platform.routing.core",
          "version": "10.10"
        },
        ...
    ]
```
Did you try to use a `SimplifiedServerInfoWriter` as suggested?
The bundles should be sorted.
What about the Warnings on Startup?